Modelica/Blocks/Continuous.mo

```diff
@@ -765,13 +765,13 @@ to compute u by an algebraic equation.
       "Control error (set point - measurement)";
     parameter .Modelica.Blocks.Types.SimpleController controllerType=
            .Modelica.Blocks.Types.SimpleController.PID "Type of controller";
-    parameter Real k(min=0) = 1 "Gain of controller";
+    parameter Real k = 1 "Gain of controller, must be non-zero";
     parameter SI.Time Ti(min=Modelica.Constants.small)=0.5
       "Time constant of Integrator block" annotation (Dialog(enable=
             controllerType == .Modelica.Blocks.Types.SimpleController.PI or
             controllerType == .Modelica.Blocks.Types.SimpleController.PID));
-    parameter SI.Time Td(min=0)=0.1
-      "Time constant of Derivative block" annotation (Dialog(enable=
+    parameter SI.Time Td(min=0)=0.1 "Time constant of Derivative block"
+                                          annotation (Dialog(enable=
```

Comment:
This change doesn't look like it should be part of this PR.

Reply:
Caused by expanding annotations in Dymola. This does not show up in local diff but is noticeable after the push. I'm working on it.

Reply:
Fixed in the new commit now.